data/at/ivb-hafas-mgate.json

```diff
@@ -13,11 +13,110 @@
     "isProprietary": true
   },
   "coverage": {
-    "regularCoverage": {
-      "region": [ "AT-7" ]
-    },
     "realtimeCoverage": {
-      "region": [ "AT-7" ]
+      "region": [ "AT-7" ],
+      "area": {
+        "type": "MultiPolygon",
+        "coordinates": [
+          [
+            [
+              [10.92, 46.77],
```

Comment:
Note that this polygon does not *cover* Tirol, it merely approximates its shape. Maybe we should change [`fill-coverage-area.py`](https://github.com/public-transport/transport-apis/blob/b1eb34fac02df532c7c61bd6ee0ea5df1faedc25/tools/fill-coverage-area.py) to add a buffer?

Reply:
Right, that is a consequence of the polygon simplification done in that script. We could counter that by doing less aggressive simplification (making the files here significantly larger) or by adding a buffer.

I'd clearly prefer the buffer (which is what we are doing in KPublicTransport already anyway, with even further simplified polygons). However:
* it will likely mean depending on some non-standard Python module for the necessary polygon math (not pretty but acceptable IMHO)
* we should try to do this consistently for all coverage polygons, so that consumers doing further processing don't add more buffers themselves than necessary. So we probably want to define if this buffer is just enough to guarantee the exact region area is fully covered, or whether it expands N kilometers beyond that to include areas for which that service is also likely relevant. I'd go for the former as the latter quickly gets use-case/application dependent.

Reply:
@1Maxnet1 What do you think we should do? Shall we vendor-in the polygon lib and add a buffer?

Reply:
Are you aware oft any official coverage information and how they communicate it? (Just curious as this problem seems to me, as someone else might already have encountered and maybe solved it)

I'd agree and add a buffer and we should clearly document that the added buffer is to guarantee that the supported region is completely covered. I would also not add the N kilometre buffer Volker mentioned.

Regarding the polygon lib for python: Imho it is fine to add it, we might want to also add a requirement. txt at this point as well, to allow users to get started quickly.

Reply:
I'm not aware of "official" sources for this. So far I think we mainly rely on the following:
* Hand-drawn oversimplified and usually significantly larger than needed polygons.
* Relatively detailed GeoJSONs found via DB Navigator internal API (https://reiseauskunft.bahn.de/addons/verbuende.json). That only covers some German operators though.
* ISO 3166-1/-2 boundary polygons from OSM. Those are very precise and thus way too large to use directly.

Reply:
https://github.com/public-transport/transport-apis/pull/72 implements adding a buffer.

Reply:
> 
    - Hand-drawn oversimplified and usually significantly larger than needed polygons.
    - Relatively detailed GeoJSONs found via DB Navigator internal API (https://reiseauskunft.bahn.de/addons/verbuende.json). That only covers some German operators though.
    - ISO 3166-1/-2 boundary polygons from OSM. Those are very precise and thus way too large to use directly.

Thanks for the info. I think we are good to go with the approach at #72 then